src/encoded/static/components/genome_browser.js

```diff
@@ -475,7 +482,12 @@ class GenomeBrowser extends React.Component {
                         <div ref={(div) => { this.chartdisplay = div; }} className="valis-browser" />
                     </div>
                 :
-                    <div className="browser-error valis-browser">There are no visualizable results.
+                    <div>
```

Comment:
With React 16, you can replace some of these `<div>`s with `<React.Fragment>` to reduce the number of levels of DOM nodes — if CSS styles within don’t rely on these `<div>`s.

Reply:
Awesome!!